sycl/source/queue.cpp

```diff
@@ -82,29 +83,47 @@ bool queue::is_host() const {
 
 void queue::throw_asynchronous() { impl->throw_asynchronous(); }
 
-event queue::memset(void *Ptr, int Value, size_t Count) {
+event queue::memset(void *Ptr, int Value,
```

Comment:
This should be an ABI breaking change that are not allowed now.

Reply:
Reverted code location information, but going ahead with notifications as this is also a heavily requested instrumentation feature.